### DISCUSSION TITLE
Dropdown and Links

### DISCUSSION THREAD

**Discussion by juan-manuel-alberro** [heroui-inc/heroui#1422] [Q&A]
Hi all,

I have the following code 

```jsx
...
<DropdownItem key="new" shortcut="⌘N">
  <Link href="/new">New</Link>
</DropdownItem>
...
```
but the click only works on the _New_ word and not in the entire row of the dropdown item.
What's the best practice for this? I know there's an `onAction` event that reads the `key` but I feel that's not really elegant when you have a complex dropdown.

Thanks!

**Comment by tetslee** [comment]
You can use the href prop from DropdownItem, also as={Link} if you're using a custom Link component.
